### DISCUSSION TITLE
How to add custom boolean methods for DateTime columns in ActiveRecord

### DISCUSSION THREAD

**Discussion by sidk** [therubyradar/submissions#151] [Submissions]
Hi there, I hope this is the right place for this. I wrote an article and would like to submit it for inclusion in the Ruby Radar newsletter: https://www.ducktypelabs.com/ar-custom-booleans/

Thanks!

**Comment by cjilbert504** [comment]
@sidk - Thank you for the submission! It'll be in today's issue!
